ios/RNCSafeAreaViewEdgeMode.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, RNCSafeAreaViewEdgeMode) {
+  RNCSafeAreaViewEdgeModeOff,
+  RNCSafeAreaViewEdgeModeAdditive,
+  RNCSafeAreaViewEdgeModeMaximum
+};
+
+@interface NSString (EnumParser)
```

Comment:
Add this, then if you import this header into `RNCSafeAreaViewEdges.m`, you'll be able to use `RCT_ENUM_CONVERTER`

```suggestion
#import <React/RCTConvert.h>

@interface RCTConvert (RNCSafeAreaViewEdgeMode)
+ (RNCSafeAreaViewEdgeMode)RNCSafeAreaViewEdgeMode:(nullable id)json;
@end
```